test/remote/gateways/remote_kushki_test.rb

```diff
@@ -51,6 +51,15 @@ def test_failed_purchase
     assert_equal 'Monto de la transacción es diferente al monto de la venta inicial', response.message
   end
 
+  def test_successful_refund
+    purchase = @gateway.purchase(@amount, @credit_card)
+    assert_success purchase
+
+    assert refund = @gateway.refund(@amount, purchase.authorization)
+    assert_success refund
+    assert_equal 'Succeeded', refund.message
+  end
+
```

Comment:
Do we want a failed refund remote test? Or does it not actually trigger in sandbox?

Reply:
I imagine if no `ticketNumber` is sent the sandbox would fail?

Reply:
I tried many ways to generate a failure response here. You would think that a failure would occur with an empty `ticketNumber`, but I only get a 500 error with message: ```{"message":"Missing Authentication Token"}%```

Reply:
Added a remote test for failed refunds.

Reply:
@dtykocki I wonder if that means we should count 500s as failure. I've seen some adapters do that.

Reply:
Yeah, treating a 500 as a failure is something I've considered, but question if that change should happen along with switching to a "real" refund. Best I can tell, all of Kushki's endpoints may return either a 400 or a 500 with a response code and message.